rust/tool/systemd/src/cli.rs

```diff
@@ -99,3 +120,48 @@ fn install(args: InstallCommand) -> Result<()> {
     )
     .install()
 }
+
+fn build(args: BuildCommand) -> Result<()> {
```

Comment:
I tried to make this simple but failed because install is very focused on install semantics
and hard to generalize.

With #204, I imagine I can simplify a lot all this stuff.

Reply:
#204 went in. Looking forward to the simpler version. :)